plugins/BEdita/API/tests/TestCase/Controller/ObjectsControllerTest.php

```diff
@@ -65,20 +65,22 @@ public function testIndex()
                     'attributes' => [
                         'status' => 'on',
                         'uname' => 'first-user',
-                        'locked' => true,
-                        'created' => '2016-05-13T07:09:23+00:00',
-                        'modified' => '2016-05-13T07:09:23+00:00',
-                        'published' => null,
                         'title' => 'Mr. First User',
                         'description' => null,
                         'body' => null,
                         'extra' => null,
                         'lang' => 'eng',
-                        'created_by' => 1,
-                        'modified_by' => 1,
                         'publish_start' => null,
                         'publish_end' => null,
                     ],
+                    'meta' => [
+                        'locked' => true,
```

Comment:
I like the solution to keep those fields in the `meta` field. An administrator can always change those properties with a custom hidden set like we already do with `password_hash`